tools/bgpreader.c

```diff
@@ -684,7 +695,7 @@ int main(int argc, char *argv[])
         if(rpki_input != NULL && rpki_input->rpki_active){
           bs_elem->annotations.cfg = cfg;
           bs_elem->annotations.rpki_active = rpki_input->rpki_active;
-          bs_elem->annotations.timestamp = bs_record->time_sec;            
+          bs_elem->annotations.timestamp = bs_record->time_sec;
```

Comment:
hmm. this is something that slipped in from rpki. why are they reproducing the timestamp here???